### ISSUE TITLE
Trouble following README install directions.

### ISSUE THREAD

**Issue by J3ffcon1** [codeforpdx/dwellingly-app#50]
Running npm install.

```
There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "babel-eslint": "10.0.3"

Don't try to install it manually: your package manager does it automatically.
However, a different version of babel-eslint was detected higher up in the tree:

  /Users/jeffreylonergan/dev/dwellingly-app/node_modules/babel-eslint (version: 7.2.3) 

Manually installing incompatible versions is known to cause hard-to-debug issues.

If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.

To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "babel-eslint" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /Users/jeffreylonergan/dev/dwellingly-app/node_modules/babel-eslint is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls babel-eslint in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed babel-eslint.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dwellingly@0.1.0 start: `react-scripts start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dwellingly@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jeffreylonergan/.npm/_logs/2020-01-16T04_04_07_462Z-debug.log
Jeffreys-MacBook-Pro:dwellingly jeffreylonergan$ npm install
npm WARN react-scripts@3.3.0 requires a peer of typescript@^3.2.1 but none is installed. You must install peer dependencies yourself.
npm WARN sass-loader@8.0.0 requires a peer of node-sass@^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN sass-loader@8.0.0 requires a peer of sass@^1.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN sass-loader@8.0.0 requires a peer of fibers@>= 3.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.

audited 905116 packages in 39.369s
found 2 moderate severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by Chris-Boe** [comment]
Looks like someone else ran into this issue on Mac: 
https://stackoverflow.com/questions/59215186/yarn-start-throws-babel-eslint-error-macos-how-do-i-fix-this-error

This example is using `yarn` instead of `npm` but it looks like the same error

**Comment by J3ffcon1** [comment]
That fixed it.

Node didn't like having 'babel-eslint' somewhere in a directory that was outside of my repository directory.  I ended up following the instructions in that stackoverflow forum post which had me search for 'babel-eslint' in my Finder for my entire PC. In finding those folders I went ahead and deleted them.

From there in going back to my repository I removed the installed node_modules folder and ran 'npm install' one more time in order to install those modules again. From there in running 'npm start' the localhost server loaded. 

Node didn't like my babel-eslint dependency that was in a different folder.
